### PR TITLE
[PPP-4447] - Upgraded jackrabbit-core to 2.16.5 to match the server a…

### PR DESCRIPTION
**Pull request by wseyler**
…nd resolve issues with Tika vulnerabilities.

### CODE REVIEW

**Comment by buildguy** [comment]
### :gear: Build finished in 2m 50s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies -pl plugins/pur/core
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|272|0|0|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6947/1/testReport)|
|----|


:information_source:  _This is an automatic message_

File: plugins/pur/core/pom.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
     <!-- Test dependencies -->
     <se-jcr.version>0.9</se-jcr.version>
     <jcr.version>2.0</jcr.version>
-    <jackrabbit-core.version>2.14.2</jackrabbit-core.version>
+    <jackrabbit-core.version>2.16.5</jackrabbit-core.version>
```
Comment (pentaho-lmartins) [inline plugins/pur/core/pom.xml]: Please move the version property to the parent pom repository and use dependency management.

**Comment by buildguy** [comment]
### :gear: Build finished in 3m 52s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -P !assemblies -pl plugins/pur/core
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|272|0|0|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6947/2/testReport)|
|----|


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 5m 28s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -pl plugins/pur/core
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|272|0|0|

|[Details](http://wingman2.pentaho.net/job/projects/job/pdi/job/PR-6947/3/testReport)|
|----|


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 55s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -pl plugins/pur/core
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by wseyler** [comment]
@pentaho-lmartins This PR can't build until the PR for maven-parent-poms is merged.

**Comment by buildguy** [comment]
### :gear: Build finished in 1m 29s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -pl plugins/pur/core
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 1m 19s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -pl plugins/pur/core
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

File: plugins/pur/core/pom.xml [hunk 2]
```diff
@@ -329,7 +328,7 @@
     <dependency>
       <groupId>org.apache.jackrabbit</groupId>
       <artifactId>jackrabbit-core</artifactId>
-      <version>${jackrabbit-core.version}</version>
+      <version>${jackrabbit.version}</version>
```
Comment (pentaho-lmartins) [inline plugins/pur/core/pom.xml]: Please use dependency management in the parent POM.
All references to versions should only exist in the parent POM.

**Comment by buildguy** [comment]
### :gear: Build finished in 1m 18s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -pl plugins/pur/core
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 1m 23s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -pl plugins/pur/core
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 1m 56s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -pl plugins/pur/core
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 1m 23s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -pl plugins/pur/core
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :gear: Build finished in 59s with command:
```console
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd \
-Daudit -pl plugins/pur/core
```
:exclamation: No tests found!


:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 1m 39s
Build command:
```console
mvn clean deploy -B -e -Daudit -pl plugins/pur/core
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 1m 57s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl plugins/pur/core
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by pentaho-lmartins** [comment]
Closing in favor of #7016

**Pull request closed without merging**